llvm/test/CodeGen/CHERI-Generic/Inputs/atomic-rmw-cap-ptr-cap-arg.ll

```diff
@@ -46,6 +46,40 @@ bb:
   ret i32 addrspace(200)* %tmp
 }
 
+; Function Attrs: nofree norecurse nounwind
```

Comment:
Will drop this commit before merging since the next commit tests the same stuff.